src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheck.java

```diff
@@ -99,6 +121,28 @@
  *   &lt;property name=&quot;allowLineBreaks&quot; value=&quot;false&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Test {
+ *   int a, b;
+ *   Test(int a, int b) {
+ *     this.
+ *         a = a; // violation, linebreak after '.' is not allowed
+ *     this.b = b; // OK
```

Comment:
Now use will confused, why config is changing, but violations are not.
Please make java examples to in such content to have on some line violation with one config and no violation with another.

Reply:
done

Reply:
Please use same java code in all examples.
To let user see how config changes result in violation changes on same code

Reply:
CLI Exceution for default config

![image](https://user-images.githubusercontent.com/70961605/168069143-60f92b5f-dd0e-45f6-ac6a-b5456dc098eb.png)

Reply:
Too much focus on violation after DOT, please add few more violation(+2 of different kind) of different kind.
Check do space validation for bunch of tokens 
https://checkstyle.org/config_whitespace.html#NoWhitespaceAfter_Properties